test/unit/self-evict-test.js

```diff
@@ -184,68 +184,72 @@ testRingpop({async: true}, 'self evict ticks membership to speed up gossip', fun
 
     assert.plan(1);
 
-    ringpop.gossip.tick = function tick(cb) {
-        assert.ok('gossip ticked!');
-        cb();
+    ringpop.client = {
+        protocolPing: function(opts, body, cb) {
+            assert.ok('gossip ticked!');
+            cb(null, {});
+        },
+        destroy: function noop() {
+        }
     };
     var selfEvict = new SelfEvict(ringpop);
     selfEvict.initiate(cleanup);
 });
 
-testRingpop({async: true}, 'self evict does not ticks membership when "selfEvictionPingEnabled" is disabled', function t(deps, assert, cleanup) {
+testRingpop({async: true}, 'self evict ping count is correct', function t(deps, assert, cleanup) {
     var ringpop = deps.ringpop;
+    ringpop.config.set('selfEvictionMaxPingRatio', 1.0);
+
     ringpop.membership.makeChange('127.0.0.1:30002', Date.now(), Member.Status.alive);
-    ringpop.config.set('selfEvictionPingEnabled', false);
+    ringpop.membership.makeChange('127.0.0.1:30003', Date.now(), Member.Status.alive);
+    ringpop.membership.makeChange('127.0.0.1:30004', Date.now(), Member.Status.faulty);
 
-    ringpop.gossip.tick = function tick(cb) {
-        assert.fail('gossip should not tick!');
-        cb();
+    assert.plan(4);
+
+    ringpop.client = {
+        protocolPing: function(opts, body, cb) {
+            var target = opts.host;
+            if (target === '127.0.0.1:30002') {
+                assert.ok('127.0.0.1:30002 is pinged!');
+                cb(null, {changes: []});
+            } else if (target === '127.0.0.1:30003') {
+                assert.ok('127.0.0.1:30003 is pinged!');
+                cb('error', null);
+            } else {
+                assert.fail('incorrect target');
```

Comment:
Is this assertion not being called? I would expect it to be called with an empty target base on my earlier comment.